projects/rangers/index.js

```diff
@@ -0,0 +1,36 @@
+const { sumTokens2 } = require('../helper/unwrapLPs');
+const {toUSDTBalances} = require('../helper/balances')
+
+const tokens = [
+  '0x71d9cfd1b7adb1e8eb4c193ce6ffbe19b4aee0db',
+  '0x8e8816a1747fddc5f8b45d2e140a425d3788f659',
+  '0x36426b7bf5709e5c2160411c6e8b1832e3404fe1',
+  '0xdaa6a6919c9543d8787490f5e9ad532c4d7ce9e8',
+  '0x3978e3cAB1c503eFaD75cB929C7076B7f4f3b6F2'
+]
+const owners = [
+  '0x00852E9CDF414D6D9403468340f78ea93Eda82d9',
```

Comment:
similarly, I am unsure of all these addresses, like what they are, can you fill in the form in PR description so we get a better idea of this new listing.

Also, staking token value goes under 'staking'

Reply:
"Tokens" is the contract address of the five tokens on the rpg chain, "RPG, USDT, MIX, AMG, COIN". "tokens" is the contract address of the Dapp platform

Reply:
"Tokens" is the contract address of the five tokens on the rpg chain, "RPG, USDT, MIX, AMG, COIN". "owners" is the contract address of the Dapp platform